data/events/SB2018/teams.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
Could we trim this file to just the data that we need?

In particular, the URLs don't make sense here.

Reply:
3c70f5984aca5744911738c46cb392237803aae0 and 5ac29fc6dafcd51fadeb02161ec36abfb27689c3 tidy this a bit.

Reply:
The idea was to keep this a pure dump from `sr-comp`, verbose or otherwise

Reply:
Why would that be useful?

Reply:
So it's easier to archive the data, and doesnt require editing the JSON in any way

Reply:
I see. I'm not sure that's a useful property.

I'd much rather have the data here be in a format most useful for use here, without tying that to the output from srcomp-http (which could change in a new version).